studio/components/interfaces/Organization/OAuthApps/OAuthApps.utils.ts

```diff
@@ -0,0 +1,42 @@
+import { createClient } from '@supabase/supabase-js'
+import { uuidv4 } from 'lib/helpers'
+
+const BUCKET_NAME = 'oauth-app-icons'
+const SUPPORT_API_URL = process.env.NEXT_PUBLIC_SUPPORT_API_URL || ''
+const SUPPORT_API_KEY = process.env.NEXT_PUBLIC_SUPPORT_ANON_KEY || ''
+
+export const uploadAttachment = async (slug: string, image: File) => {
+  const supabaseClient = createClient(SUPPORT_API_URL, SUPPORT_API_KEY, {
+    auth: {
+      persistSession: false,
+      autoRefreshToken: false,
+      // @ts-ignore
+      multiTab: false,
+      detectSessionInUrl: false,
+      localStorage: {
+        getItem: (key: string) => undefined,
+        setItem: (key: string, value: string) => {},
+        removeItem: (key: string) => {},
+      },
+    },
+  })
```

Comment:
Should this be in a util function or something? Feels like it's reusable?

Reply:
you're right - i'll add this as a method in the lib folder, can be reused in multiple places including support form and feedback attachments

Reply:
but i'll just refactor here first